controller.go

```diff
@@ -598,7 +603,11 @@ func (c *Controller) updateService(tunnel *inletsv1alpha1.Tunnel, ip string) err
 	// copy.Status.LoadBalancer.Ingress = []corev1.LoadBalancerIngress{
 	// 	corev1.LoadBalancerIngress{IP: ip},
 	// }
-	copy.Spec.ExternalIPs = []string{ip}
+	if ip == "" {
```

Comment:
I guess that we should only remove the one we added, since you can have > 1 IP, especially if you use k3s which has a "servicelb" that allocates ports on the hosts in the cluster.

Reply:
Yes I totally agree.. 

I'm not really sure what is happening here, but at first, I implemented a way to remove just the ip from `tunnel.host.ip`. It worked, but then I realized that to add the ip the code it's also "overwriting" all the ips. 

In my tests with k3s and metallb it works fine. I have a service with ips from traefik and metallb and those ips are not being removed.

Unless there is something that is maintaining the state of the ips from  trefik and metallb.

Anyway makes sense to change the code to remove only the IP from the tunnel.